main.py

```diff
@@ -53,7 +54,11 @@ def main(POST_ID=None) -> None:
     bg_config = get_background_config()
     download_background(bg_config)
     chop_background_video(bg_config, length, reddit_object)
-    make_final_video(number_of_comments, length, reddit_object, bg_config)
+    try:
+        make_final_video(number_of_comments, length, reddit_object, bg_config)
+    except ffmpeg.Error as e:
+        print(e.stderr.decode("utf8"))
+        exit(1)
```

Comment:
please keep this function clean

Reply:
I agree with you, but this is the easiest way

Reply:
i think by just watching where this exception can be raised and putting this try-except would work